fil_actors_shared/src/actor_versions.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019-2025 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use serde::{Deserialize, Serialize};
+
+/// Enum representing supported actor versions.
+#[derive(
+    Debug,
+    Clone,
+    Copy,
+    PartialEq,
+    Eq,
+    Hash,
+    PartialOrd,
+    Ord,
+    Serialize,
+    Deserialize,
+    strum::Display,
+    strum::EnumString,
+    strum::FromRepr,
+)]
+#[repr(u8)]
+pub enum ActorVersion {
+    #[strum(to_string = "v8")]
```

Comment:
do we actually need this? Wouldn't `#[strum(serialize_all = "lowercase")]` work?

Reply:
Done.